internal/services/pam/pam.go

```diff
@@ -190,6 +190,50 @@ func (s Service) SetDefaultBrokerForUser(ctx context.Context, req *authd.SDBFURe
 	return &authd.Empty{}, nil
 }
 
+func (s Service) AbortSession(ctx context.Context, req *authd.ASRequest) (empty *authd.Empty, err error) {
```

Comment:
Still unsure where to link this in `pam_sm_authenticate`...

Reply:
No need to have is properly linked yet. This will be properly on Ctrl+C cleanup, but that’s once we have converted to the new UI!